setup.py

```diff
@@ -52,6 +52,7 @@
         ],
     },
     install_requires=[
+        'flaky',
         'pexpect',
         'plumbum',
         'pytest',
```

Comment:
pytest-selenium is not used anymore. Maybe worth removing it from this list.

Reply:
removed now.